net/mmfd/files/lib/gluon/upgrade/430-add-mmfd-interface

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/lua
+
+local uci = require('luci.model.uci').cursor()
+
+uci:delete('network', 'mmfd')
+uci:section('network', 'interface', 'mmfd', {
+	proto = 'static',
+	ifname = 'mmfd0',
+	ip6addr = 'fe80::1/64'
+}
+)
+uci:save('network')
```

Comment:
I think it would be nice if mmfd would just setup its address itself, without relying on netifd.